src/index.html

```diff
@@ -129,6 +138,19 @@ <h2 class="sidebar-header-name" id="sidebarUserName" tabindex="0">Name Name</h2>
           </section>
         </section>
       </section>
+      <section class="activity-input">
+        <label>User ID:</label>
+        <input id="activity-userID-input" type="number">
+        <label>Date:</label>
+        <input id="activity-date-input" type="text">
+        <label>Number of Steps:</label>
+        <input id="activity-numSteps-input" type="number">
+        <label>Minutes Active:</label>
+        <input id="activity-minsActive-input" type="number">
+        <label>Flights of Stairs:</label>
+        <input id="activity-flightsOfStairs-input">
+        <button id="submit-activity-info">Submit</button>
+      </section>
```

Comment:
Glad to have gotten these inputs set up - we'll come back around to fix up details on attributes (as necessary) and styling later on